src/features/profile/Profile.test.tsx

```diff
@@ -221,18 +221,6 @@ describe('Profile related components', () => {
     });
   });
 
-  test('renders ProfileWrapper with auth message when missing auth', () => {
-    render(
-      <Provider store={createTestStore()}>
-        <Router initialEntries={[`/profile/${usernameRequested}`]}>
-          <Routes />
-        </Router>
-      </Provider>
-    );
-    const linkElement = screen.getByText(/Loading authentication state/i);
-    expect(linkElement).toBeInTheDocument();
-  });
-
```

Comment:
This test was removed because `Loading authentication state` doesn't get displayed due to the `Authed`-controlled redirect. I kept the auth fallback in the component to satisfy typescript, and it gets coverage by other tests.